packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.test.ts

```diff
@@ -167,10 +167,11 @@ const nonEmptyResponseTestCases = Object.entries(testMap) as Array<
 
 // Endpoints returning void response on success
 const emptyResponseTestCases = [
-    deleteTestCase,
     addPriceBooksToBasketTestCase,
     addTaxesForBasketTestCase,
-    addTaxesForBasketItemTestCase
+    addTaxesForBasketItemTestCase,
+    // FIXME: This test only passed it fun last.
```

Comment:
```suggestion
    // FIXME: This test only passed if run last.
```
Is that what you meant?